### ISSUE TITLE
nan loss

### ISSUE THREAD

**Issue by CuongNN218** [FangShancheng/ABINet#13]
![image](https://user-images.githubusercontent.com/26113323/124248045-a9574b80-db4c-11eb-9b5c-e8a1f5b8a8d9.png)
Hi, thanks for your great work @FangShancheng 
I try to training ABINet with vietnamese characters. Firstly, I train the language model using my own vietnamese token but got nan loss. Can you help me to fix this ?

**Comment by CuongNN218** [comment]
CUDA_VISIBLE_DEVICES=0,1,2,3 python main.py --config=configs/pretrain_language_model.yaml
I using this cmd to train my language model.

**Comment by CuongNN218** [comment]
[train.txt](https://github.com/FangShancheng/ABINet/files/6753400/train.txt)
Here is my train.txt

**Comment by CuongNN218** [comment]
I found a line of code is not suitable for my charset. I will close this topic. thanks

**Comment by FangShancheng** [comment]
Also, if text with a length small than 2 (i.e, length 1) will cause NAN, due to masking all tokens.

**Comment by CuongNN218** [comment]
@FangShancheng how long did you spent to train the language model ?

**Comment by FangShancheng** [comment]
stop training if the loss curve converges

**Comment by archwolf118** [comment]
> I found a line of code is not suitable for my charset. I will close this topic. thanks

Dear CuongNN218,
Which line should be fix?
I encounter same problem, and I got the Nan result.
Thank you!

**Comment by hungcao0402** [comment]
I meet the same problem, how did you fix it?
